README.md

```diff
@@ -122,6 +134,10 @@ The phase is updated for each processing attempt on the CR.
 1. Deploy the operator as explained above
 2. run `make cluster-functest`
 
+### Debug with must-gather
```

Comment:
Maybe a new subsection for must-gather is better then putting it under `Tests`

Reply:
agreed